tokens/governance/treasury-spend.md

```diff
@@ -1,97 +1,103 @@
+    [0x6d6F646c70632f74727372790000000000000000](https://moonbase.subscan.io/account/0x6d6F646c70632f74727372790000000000000000){target=_blank}
+
 
 ## Roadmap of a Treasury Proposal {: #roadmap-of-a-treasury-proposal }
 
-You can find a full explanation of the [happy path for a Treasury proposal on the Treasury overview page](/learn/features/treasury/){target=\_blank}. For more information on the process, see this [guide to the interim community Treasury program](https://moonbeam.network/news/proposal-for-interim-treasury-program-approved-by-moonbeam-community){target=\_blank}.
+The happy path of a treasury spend request is as follows:
 
-![Proposal Roadmap](/images/tokens/governance/treasury-proposals/treasury-proposal-roadmap.webp)
+1. **Proposal Submission** - The user submits a proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}
 
-## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+2. **Forum Discussion** - The proposal is discussed by the community on the Forum. The ultimate yay/nay decision is determined by the treasury council
 
-It's highly recommended that you preface any proposal with a post on [Moonbeam's Community Forum](https://forum.moonbeam.network){target=\_blank}. You should allow a period of five days for the community to provide feedback on the Moonbeam Forum post. This is especially important for Treasury proposals because of the irrevocable bond required to submit a Treasury spend proposal. You can submit your idea for a Treasury spend proposal by following [the instructions in the Using the Moonbeam Community Forum to Submit a Treasury Proposal guide](https://moonbeam.network/news/using-the-moonbeam-community-forum-to-submit-a-treasury-proposal){target=\_blank}.
+3. **Treasury Approval & Action** - If the Treasury Council agrees, it authorizes the treasury spending and moves the process forward
 
 ![Moonbeam Forum Home](/images/tokens/governance/treasury-proposals/treasury-proposal-1.webp)
 
-## Proposing an Action {: #proposing-an-action }
+## Submitting your Idea to the Forum {: #submitting-your-idea-to-the-forum }
+
+As mentioned in the happy path above, the first step of a treasury proposal is to submit the proposal to the [Moonbeam Forum](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}. To properly submit a treasury spend request, take the following steps:
+
+1. From the [governance section](https://forum.moonbeam.network/c/governance/treasury-proposals/8){target=\_blank}, click **New Topic**. By starting the topic in the governance section, the proposal will come pre-filled with a template to ensure you cover all the necessary points 
```

Comment:
Governance section is something that you're pointing out and they're looking at in the UI, so **Governance** should be bold and capitalized as it is on the forum